code/modules/mob/living/silicon/silicon_defense.dm

```diff
@@ -115,21 +115,28 @@
 			M.visible_message(span_boldwarning("[M] is thrown off of [src]!"))
 	flash_act(affect_silicon = 1)
 
-/mob/living/silicon/bullet_act(obj/projectile/Proj, def_zone, piercing_hit = FALSE)
-	SEND_SIGNAL(src, COMSIG_ATOM_BULLET_ACT, Proj, def_zone)
-	if((Proj.damage_type == BRUTE || Proj.damage_type == BURN))
-		adjustBruteLoss(Proj.damage)
-		if(prob(Proj.damage*1.5))
-			for(var/mob/living/M in buckled_mobs)
-				M.visible_message(span_boldwarning("[M] is knocked off of [src]!"))
-				unbuckle_mob(M)
-				M.Paralyze(40)
-	if(Proj.stun || Proj.knockdown || Proj.paralyze)
-		for(var/mob/living/M in buckled_mobs)
-			unbuckle_mob(M)
-			M.visible_message(span_boldwarning("[M] is knocked off of [src] by the [Proj]!"))
-	Proj.on_hit(src, 0, piercing_hit)
-	return BULLET_ACT_HIT
+/mob/living/silicon/bullet_act(obj/projectile/hitting_projectile, def_zone, piercing_hit = FALSE)
+	// A bit cringe but silicons are immune to these effects on projectiles
+	hitting_projectile.slur = 0 SECONDS
```

Comment:
This will make projectiles hitting multiple targets weird. Should probably be moved up with some inherent silicon trait.